### PR TITLE
Correct description of `ParticleProcessMaterial.lifetime_randomness`

### LINKED ISSUE #7559: Possible incorrect description for lifetime_randomness
**Your Godot version:**
v4.1.beta3.official [ada712e06]

**Issue description:**
It says it scales from value to 1.0, but from what I see it scales from (1.0 - value) to 1.0. If value is 0.4, it will scale from (1.0 - 0.4), that is from 0.6 to 1.0, not from 0.4. If I set it to 1.0, it will scale from 0.0 (1.0 - 1.0) to 1.0, and default value is 0.0, which doesn't affect lifetime at all.

**URL to the documentation page (if already existing):**
[**Docs.**](https://docs.godotengine.org/en/stable/classes/class_particleprocessmaterial.html#class-particleprocessmaterial-property-lifetime-randomness)

### PR DESCRIPTION
**Pull request by CURR-E**
The current description fails to correctly explain the equation which calculates a particle's lifetime.

The new description explains the equation properly.

Fixes [godot/godot-docs/#7559](https://github.com/godotengine/godot-docs/issues/7559#issue-1773652586)

### CODE REVIEW

**Comment by AThousandShips** [comment]
This might be better served by showing the equation instead of describing it in words, it gets quite difficult to work out what it does

Also please update your commit message to be more descriptive, like how I changed the title of this PR

**Comment by AThousandShips** [comment]
I think the best way is to first squash, and then use "git commit --amend" to update the message

**Comment by CURR-E** [comment]
After a lot of confusion, I think it's properly squashed and amended now... hopefully.

**Review by clayjohn — APPROVED** [review]

File: doc/classes/ParticleProcessMaterial.xml [hunk 1]
```diff
@@ -209,7 +209,7 @@
 			Minimum equivalent of [member initial_velocity_max].
 		</member>
 		<member name="lifetime_randomness" type="float" setter="set_lifetime_randomness" getter="get_lifetime_randomness" default="0.0">
-			Particle lifetime randomness ratio. The lifetime will be multiplied by a value interpolated between [code]1.0[/code] and a random number less than one. For example a random ratio of [code]0.4[/code] would scale the original lifetime between [code]0.4-1.0[/code] of its original value.
+			Particle lifetime randomness ratio. The equation for the lifetime of a particle is [code]lifetime * (1.0 - randf() * lifetime_randomness)[/code]. For example a lifetime_randomness of [code]0.4[/code] scales the lifetime between [code]0.6[/code] to [code]1.0[/code] of its original value.
```
Comment (Calinou) [inline doc/classes/ParticleProcessMaterial.xml:212]:
```suggestion
			Particle lifetime randomness ratio. The equation for the lifetime of a particle is [code]lifetime * (1.0 - randf() * lifetime_randomness)[/code]. For example, a [member lifetime_randomness] of [code]0.4[/code] scales the lifetime between [code]0.6[/code] to [code]1.0[/code] of its original value.
```

**Comment by CURR-E** [comment]
Oops, I think that push I just made was unchanged. Trying again.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Pull request merged**